system-variables.md

```diff
@@ -616,34 +643,36 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 ### tidb_expensive_query_time_threshold
 
 - Scope: INSTANCE
-- Default value: 60
+- Default value: `60`
+- Range: `[10, 2147483647]`
 - This variable is used to set the threshold value that determines whether to print expensive query logs. The unit is second. The difference between expensive query logs and slow query logs is:
     - Slow logs are printed after the statement is executed.
     - Expensive query logs print the statements that are being executed, with execution time exceeding the threshold value, and their related information.
 
 ### tidb_force_priority
 
 - Scope: INSTANCE
-- Default value: NO_PRIORITY
+- Default value: `NO_PRIORITY`
 - This variable is used to change the default priority for statements executed on a TiDB server. A use case is to ensure that a particular user that is performing OLAP queries receives lower priority than users performing OLTP queries.
 - You can set the value of this variable to `NO_PRIORITY`, `LOW_PRIORITY`, `DELAYED` or `HIGH_PRIORITY`.
 
 ### tidb_gc_concurrency <span class="version-mark">New in v5.0</span>
 
 - Scope: GLOBAL
-- Default: -1
+- Default value: `-1`
+- Range: `[1, 128]`
 - Specifies the number of threads in the [Resolve Locks](/garbage-collection-overview.md#resolve-locks) step of GC. A value of `-1` means that TiDB will automatically decide the number of garbage collection threads to use.
 
 ### tidb_gc_enable <span class="version-mark">New in v5.0</span>
 
 - Scope: GLOBAL
-- Default value: ON
+- Default value: `ON`
 - Enables garbage collection for TiKV. Disabling garbage collection will reduce system performance, as old versions of rows will no longer be purged.
 
 ### tidb_gc_life_time <span class="version-mark">New in v5.0</span>
 
 - Scope: GLOBAL
-- Default: `"10m0s"`
+- Default value: `10m0s`
```

Comment:
```suggestion
- Default value: `10m0s`
- Range: `[10m, 2562047h]`
```

```sql
5.7.25-TiDB-v5.2.0-alpha-16-gf81ef5579-dirty 127.0.0.1:4000  SQL  set global tidb_gc_life_time='10m';
Query OK, 0 rows affected (0.0031 sec)
5.7.25-TiDB-v5.2.0-alpha-16-gf81ef5579-dirty 127.0.0.1:4000  SQL  set global tidb_gc_life_time='9m';
ERROR: 1232 (42000): Incorrect argument type to variable 'tidb_gc_life_time'
5.7.25-TiDB-v5.2.0-alpha-16-gf81ef5579-dirty 127.0.0.1:4000  SQL  set global tidb_gc_life_time='9m59s';
ERROR: 1232 (42000): Incorrect argument type to variable 'tidb_gc_life_time'
5.7.25-TiDB-v5.2.0-alpha-16-gf81ef5579-dirty 127.0.0.1:4000  SQL  set global tidb_gc_life_time='9m60s';
Query OK, 0 rows affected (0.0029 sec)
```

Reply:
I'm going to take this up with the team responsible before fixing it. The range here is 292 years, which doesn't really make sense so Golang formats this in minutes with duration.String() method.

I think setting it in code to be a maximum of 1 year is the logical way to handle it.